mercurial coverage build broken because of missing /corpus/xdiff_fuzzer.zip?

See https://oss-fuzz-build-logs.storage.googleapis.com/log-ecc3384e-9aad-44e4-ae67-286e8551188c.txt for example - I've gone through the reproducing steps and I can't figure out why things are sad about this zip - I don't provide a corpus for the xdiff fuzzer, and listing the GCS buckets shows some stuff, but I think this might be environmental somehow in oss-fuzz? Is there anything else I should try on my end?

I thought the following patch would fix the remaining issues:

```diff
diff --git a/projects/mercurial/Dockerfile b/projects/mercurial/Dockerfile
index 890c5237..09b52209 100644
--- a/projects/mercurial/Dockerfile
+++ b/projects/mercurial/Dockerfile
@@ -18,7 +18,7 @@ FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER security@mercurial-scm.org
 RUN apt-get update && apt-get install -y make autoconf automake libtool \
   python-dev mercurial curl
-RUN cd / && curl https://www.python.org/ftp/python/2.7.15/Python-2.7.15.tgz | tar xzf -
+RUN cd $SRC && curl https://www.python.org/ftp/python/2.7.15/Python-2.7.15.tgz | tar xzf -
 RUN hg clone https://www.mercurial-scm.org/repo/hg mercurial
 WORKDIR mercurial
 COPY build.sh $SRC/
diff --git a/projects/mercurial/build.sh b/projects/mercurial/build.sh
index 58d0670d..28ace9a4 100755
--- a/projects/mercurial/build.sh
+++ b/projects/mercurial/build.sh
@@ -15,7 +15,7 @@
 #
 ################################################################################
 
-pushd /Python-2.7.15/
+pushd $SRC/Python-2.7.15/
 ls
 patch -p1 <<'EOF'
 Index: v2_7_unstable/Python/pymath.c
@@ -50,4 +50,5 @@ popd
 
 cd contrib/fuzz
 export PYLDFLAGS=$(echo $CFLAGS | xargs -n 1 echo | egrep -- '-fsanitize=(memory|address)')
+make clean
 make oss-fuzz
```

But apparently some of the underlying scripts rely on the `/Python-2.7.15/` path:

```
/bin/sh: 1: cd: can't cd to /Python-2.7.15/
/bin/sh: 1: ./configure: not found
Makefile:80: recipe for target '/out/sanpy/bin/python' failed
make: *** [/out/sanpy/bin/python] Error 127
Fuzzers build failed.
```

if you could fix those, I'm quite confident we'd get the coverage reports.

--- a/projects/mercurial/Dockerfile
+++ b/projects/mercurial/Dockerfile
@@ -18,7 +18,7 @@ FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER security@mercurial-scm.org
 RUN apt-get update && apt-get install -y make autoconf automake libtool \
   python-dev mercurial curl
-RUN cd / && curl https://www.python.org/ftp/python/2.7.15/Python-2.7.15.tgz | tar xzf -
+RUN cd $SRC && curl https://www.python.org/ftp/python/2.7.15/Python-2.7.15.tgz | tar xzf -
 RUN hg clone https://www.mercurial-scm.org/repo/hg mercurial
 WORKDIR mercurial
 COPY build.sh $SRC/

--- a/projects/mercurial/build.sh
+++ b/projects/mercurial/build.sh
@@ -15,8 +15,7 @@
 #
 ################################################################################
 
-pushd /Python-2.7.15/
-ls
+pushd $SRC/Python-2.7.15/
 patch -p1 <<'EOF'
 Index: v2_7_unstable/Python/pymath.c
 ===================================================================
@@ -46,8 +45,19 @@ Index: v2_7_unstable/Modules/_ctypes/callproc.c
      avalues = (void **)alloca(sizeof(void *) * argcount);
      atypes = (ffi_type **)alloca(sizeof(ffi_type *) * argcount);
 EOF
+
+if [ -e $OUT/sanpy/cflags -a "$(cat $OUT/sanpy/cflags)" = "${CFLAGS}" ] ; then
+    echo 'Python cflags unchanged, no need to rebuild'
+else
+    rm -rf $OUT/sanpy
+    ASAN_OPTIONS=detect_leaks=0 ./configure --without-pymalloc \
+                --prefix=$OUT/sanpy CFLAGS="${CFLAGS}" LINKCC="${CXX}" \
+                LDFLAGS="${CXXFLAGS}"
+    grep -v HAVE_GETC_UNLOCKED < pyconfig.h > tmp && mv tmp pyconfig.h
+    ASAN_OPTIONS=detect_leaks=0 make && make install
+    echo "${CFLAGS}" > $OUT/sanpy/cflags
+fi
 popd
 
 cd contrib/fuzz
-export PYLDFLAGS=$(echo $CFLAGS | xargs -n 1 echo | egrep -- '-fsanitize=(memory|address)')
-make oss-fuzz
+make clean oss-fuzz

